.npmignore

```diff
@@ -1,3 +1,4 @@
 # Exclude everything but the contents of the dist directory.
 **/*
 !dist/**
+!src/css/**
```

Comment:
We do want the font folder as well. I, for instance, immediately copied over the font folder via grunt-contrib-copy. Dist should contain everything necessary to self-host a player.

On a different note (that shouldn't he handled here), we should consider renaming this folder `scss`.